src/llama-context.cpp

```diff
@@ -620,7 +620,7 @@ float * llama_context::get_embeddings_ith(int32_t i) {
             throw std::runtime_error(format("corrupt output buffer (j=%" PRId64 ", n_outputs=%d)", j, n_outputs));
         }
 
-        return embd + j*model.hparams.n_embd;
+        return embd + j*model.hparams.n_embd_inp();
```

Comment:
Technically this should be `n_embd_out` (aka currently `n_embd`), right?

Reply:
My understanding is this should be `n_embd` now and potentially `n_embd_out` if we do the extension.